src/components/pages/import.js

```diff
@@ -1,152 +1,149 @@
-
-    try {
-      const { documents } = await importApiClient.getImports(newSelectedSource.hostName);
-      setImportableDocuments(documents);
-    } catch (error) {
-      handleApiError({ error, t });
+  const renderBatchStatus = ({ progress }) => {
+    if (progress === 1) {
+      return <span>{t('batchStatusDone')}</span>;
+    }
+    if (progress === 0) {
+      return <span>{t('batchStatusPending')}</span>;
     }
-    setIsFetchingImportableDocuments(false);
+    return <span>{t('batchStatusProcessing', { progress: batch.progress })}</span>;
   };
 
-  const renderImportType = doc => {
+  const renderDate = date => <span>{formatDate(date)}</span>;
+  const renderTitle = ({ title, slug }) => (
+    <a
+      target="_blank"
+      href={getImportedArticleUrl({ hostName, allowUnsecure, slug })}
+      rel="noreferrer"
+      >{title}
+    </a>
+  );
+
+  const renderImportType = taskParams => {
     let icon = null;
-    if (doc.importType === DOCUMENT_IMPORT_TYPE.add) {
+    if (taskParams.importType === DOCUMENT_IMPORT_TYPE.add) {
       icon = <CloudDownloadOutlined />;
     }
-    if (doc.importType === DOCUMENT_IMPORT_TYPE.update) {
+    if (taskParams.importType === DOCUMENT_IMPORT_TYPE.update) {
       icon = <CloudSyncOutlined />;
     }
-    return <Tooltip title={t(doc.importType)} className="ImportPage-importType">{icon}</Tooltip>;
-  };
-
-  const renderTitle = doc => {
-    if (!doc.slug) {
-      return <span>{doc.title}</span>;
-    }
-
-    const baseUrl = getImportSourceBaseUrl(selectedSource);
-
-    const url = `${baseUrl}${getArticleUrl(doc.slug)}`;
-    return <a href={url} target="_blank" rel="noopener noreferrer" >{doc.title}</a>;
+    return <Tooltip title={t(taskParams.importType)} className="CreateImportPage-importType">{icon}</Tooltip>;
   };
 
-  const renderLanguage = doc => {
-    const languagesData = languageNameProvider.getData(language);
-    const documentLanguageData = languagesData[doc.language];
-    return <CountryFlagAndName code={documentLanguageData.flag} name={documentLanguageData.name} flagOnly />;
-  };
+  const renderErrors = errors => <span>{errors.join(';')}</span>;
+  const attemtpsColumns = [
+    { title: t('startedOn'), dataIndex: 'startedOn', width: '50px', render: renderDate },
+    { title: t('completedOn'), key: '_id', dataIndex: 'completedOn', width: '150px', render: renderDate },
+    { title: t('errors'), dataIndex: 'errors', width: '150px', render: renderErrors }
+  ];
 
-  const renderUpdateDate = doc => {
-    return <span>{formatDate(doc.updatedOn)}</span>;
+  const header = () => <span>{t('attemptsHeader')}</span>;
+  const getErrorsExtra = ({ errors }) => <span>{errors?.length || 0} {t('errors')}</span>;
+
+  const attemptsRenderer = (attempts, task) => {
+    const errors = task.attempts.map(attempt => attempt.errors).flat();
+    return (
+      <Collapse>
+        <Panel key={task._id} header={header()} extra={getErrorsExtra({ errors })}>
+          <Table
```

Comment:
I wanted to use an expandable table here, but the expand button wouldn't appear... So I went the collapse panel variant, not bad but not great either.